#!/usr/bin/env python
PACKAGE = "lane_detect"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("use_birdview",   bool_t,   0, "Use birdview or hough to detect line",  True)
gen.add("init_lane_width",       int_t,    0, "InitLaneWidth when detect", 50,  0, 200)
gen.add("birdwidth",       int_t,    0, "Birdview width", 320,  0, 500)
gen.add("birdheight",      int_t,    0, "Birdview height", 240,  0, 500)
gen.add("skyline",         int_t,    0, "Skyline", 120,  0, 240)
gen.add("offset_left",          int_t,    0, "Offset bottom left birdview", 100,  0, 160)
gen.add("offset_right",         int_t,    0, "Offset bottom right birdview", 100,  0, 160)

exit(gen.generate(PACKAGE, "lane_detect_node", "lane"))